src/vm/jithelpers.cpp

```diff
@@ -3028,6 +3028,106 @@ HCIMPLEND
 //
 //========================================================================
 
+//*************************************************************
+// Array allocation fast path for arrays of value type elements
+//
+
+HCIMPL2(Object*, JIT_NewArr1VC_MP_Portable, CORINFO_CLASS_HANDLE arrayTypeHnd_, INT_PTR size)
```

Comment:
Maybe we should call it `JIT_NewArr1VC_MP_FastPortable` to make it better describe what it does? It is different pattern from other _Portable JIT helpers.